src/ext/debug_backtrace.c

```diff
@@ -0,0 +1,39 @@
+#include <execinfo.h>
+#include <php.h>
+#include <signal.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+
+#include "ddtrace.h"
+#include "debug_backtrace.h"
+
+ZEND_EXTERN_MODULE_GLOBALS(ddtrace);
+
+void ddtrace_backtrace_handler(int sig) {
+    fprintf(stderr, "Datadog PHP Trace extension (DEBUG MODE)\n");
```

Comment:
What are your thoughts on using `php_log_err()` for these lines? :)
```suggestion
    php_log_err("Datadog PHP Trace extension (DEBUG MODE)");
```

Reply:
👍 good idea